src/main/groovy/info/solidsoft/gradle/pitest/PitestPlugin.groovy

```diff
@@ -170,19 +170,20 @@ class PitestPlugin implements Plugin<Project> {
         task.sourceDirs.setFrom(extension.mainSourceSets.map { mainSourceSet -> mainSourceSet*.allSource*.srcDirs })
         task.detectInlinedCode.set(extension.detectInlinedCode)
         task.timestampedReports.set(extension.timestampedReports)
+        Callable<Set<File>> allMutableCodePaths = {
+            calculateBaseMutableCodePaths() + (extension.additionalMutableCodePaths.getOrElse([] as Set) as Set<File>)
+        }
         task.additionalClasspath.setFrom({
             List<FileCollection> testRuntimeClasspath = (extension.testSourceSets.get() as Set<SourceSet>)*.runtimeClasspath
             FileCollection combinedTaskClasspath = project.objects.fileCollection().from(testRuntimeClasspath)
             FileCollection filteredCombinedTaskClasspath = combinedTaskClasspath.filter { File file ->
-                !extension.fileExtensionsToFilter.getOrElse([]).find { extension -> file.name.endsWith(".$extension") }
-            }
+                !extension.fileExtensionsToFilter.getOrElse([]).find { extension -> file.name.endsWith(".$extension") || file == "core.jar" }
```

Comment:
Why `core.jar` is special?

Reply:
I forgot to remove it after debugging a local project.